aiida/cmdline/params/options/commands/code.py

```diff
@@ -97,3 +90,29 @@ def is_not_on_computer(ctx):
     prompt_fn=is_on_computer,
     help='Name of the computer, on which the code is installed.'
 )
+
+PREPEND_TEXT = OverridableOption(
+    '--prepend-text',
+    cls=TemplateInteractiveOption,
+    prompt='Prepend script',
+    type=click.STRING,
+    default='',
+    help='Bash code that should be prepended before the executable call in all submit scripts for this code.',
```

Comment:
While "bash code" is correct as well, I would lean slightly towards "bash script" (or "bash commands"), just because code has such a special meaning in AiiDA.

And, I guess, strictly speaking it would be "prepended to".

I leave the decision up to you (both are minor suggestions, not required).

Reply:
Agree with the suggestions. Have adapted them.